draft-duke-quic-protected-initial.md

```diff
@@ -313,13 +291,28 @@ If the client does not receive a public_key_failed transport parameter in
 response to sending a public_key_failed transport parameter, it MUST terminate
 the connection with a TRANSPORT_PARAMETER_ERROR.
 
+The client MUST NOT include the original client hello in its TLS transcript
+hash for key computation, as the server has no access to this message.
+
+# The Fallback Packet {#fallback-packet}
+
+The Fallback packet uses the 0x1 packet type code, which it shares with 0RTT.
+Since 0RTT is only sent by clients and Fallback is only sent by servers, these
+two types are distinguished by the endpoint's role in the handshake.
+
+The packet is encrypted with Initial keys derived from the following well-known
+"fallback salt" 0xbd62319ad6eeb17a9ed0d3bf75e37e4a8e7e6ac7, instead of the
+shared secret that the server cannot decode.
+
+The format of this packet is TBD. However, it contains a hash of the entire
```

Comment:
I'd suggest just using GMAC like we did for the retry integrity tag, and then just having that be the full contents of the packet (no packet number or length)

Reply:
Filed #28